library/kani_core/src/mem.rs

```diff
@@ -164,8 +167,15 @@ macro_rules! kani_mem {
         }
 
         /// Checks that `ptr` points to an allocation that can hold data of size calculated from `T`.
-        ///
```

Comment:
Please add this line back (see https://rust-lang.github.io/rust-clippy/master/index.html#too_long_first_doc_paragraph)